### PR TITLE
using Q script for AI samples on Devcloud

### PR DESCRIPTION
**Pull request by louie-tsai**
# Existing Sample Changes
## Description
Make DevCloud usage on AI samples more user friendly, and could submit a task to a specific Intel Architecture such as ICX.

## External Dependencies

NA

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [ X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ X] Command Line

### CODE REVIEW

**Comment by louie-tsai** [comment]
@bdmoore1 

could you help me to review it?
thanks

File: AI-and-Analytics/DevCloudREADME.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Running Samples In DevCloud 
+### Run Samples in Jupyter Lab
+1.	Open [Intel DevCloud](https://software.intel.com/content/www/us/en/develop/tools/devcloud.html).
+2.	In the upper right corner, click Sign In.
+3.	Log in with your Intel account username and password.
+4.	Open Jupyter lab: https://jupyter.oneapi.devcloud.intel.com/
```
Comment (bdmoore1) [inline AI-and-Analytics/DevCloudREADME.md:6]: Change the jupyter link to: https://jupyter.oneapi.devcloud.intel.com/user/u80183/lab/tree/Welcome.ipynb

File: AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md [hunk 2]
```diff
@@ -99,4 +76,21 @@ In order to run on the DevCloud, you need to request a compute node using node p
     Please find more information about the mkldnn log [here](https://oneapi-src.github.io/oneDNN/dev_guide_verbose.html).
 
 ## Example of Output
-With successful execution, it will print out `[CODE_SAMPLE_COMPLETED_SUCCESSFULLY]` in the terminal.
+With successful execution, it will print out `[CODE_SAMPLE_COMPLETED_SUCCESSFULLY]` in the terminal.
```
Comment (bdmoore1) [inline AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md:79]: Above line 63 (Step 4) we need a new step.
4. Navigate to the PyTorch sample:

```
cd ~/oneAPI-samples/AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted
```

File: AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md [hunk 3]
```diff
@@ -99,4 +76,21 @@ In order to run on the DevCloud, you need to request a compute node using node p
     Please find more information about the mkldnn log [here](https://oneapi-src.github.io/oneDNN/dev_guide_verbose.html).
 
 ## Example of Output
-With successful execution, it will print out `[CODE_SAMPLE_COMPLETED_SUCCESSFULLY]` in the terminal.
+With successful execution, it will print out `[CODE_SAMPLE_COMPLETED_SUCCESSFULLY]` in the terminal.  
+
```
Comment (bdmoore1) [inline AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md:80]: After line 67 "pytorch PyTorch_Hello_World.py", I get this error message:
Traceback (most recent call last):
  File "PyTorch_Hello_World.py", line 12, in <module>
    import torch
ImportError: No module named torch

Do you know what I am missing?

File: AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md [hunk 4]
```diff
@@ -99,4 +76,21 @@ In order to run on the DevCloud, you need to request a compute node using node p
...
 ## Example of Output
-With successful execution, it will print out `[CODE_SAMPLE_COMPLETED_SUCCESSFULLY]` in the terminal.
+With successful execution, it will print out `[CODE_SAMPLE_COMPLETED_SUCCESSFULLY]` in the terminal.  
+
+
+## Running The Sample In DevCloud (Optional)
+
+Please refer to [using samples in DevCloud](https://github.com/intel-ai-tce/oneAPI-samples/blob/devcloud/AI-and-Analytics/README.md#using-samples-in-intel-oneapi-devcloud) for general usage instructions.
+
+### Submit The Sample in Batch Modeon
```
Comment (bdmoore1) [inline AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md:86]: Should "Modeon" be "Mode"?

File: AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md [hunk 5]
```diff
@@ -121,141 +121,22 @@ dnnl_verbose,exec,cpu,convolution,jit:avx512_common,forward_training,src_f32::bl
...
-dnnl_verbose,info,cpu,isa:Intel AVX-512 with Intel DL Boost
-dnnl_verbose,info,gpu,runtime:none
-dnnl_verbose,exec,cpu,reorder,jit:uni,undef,src_f32::blocked:acdb:f0 dst_f32::blocked:abcd:f0,,,4x4x128x128,12.0649
-dnnl_verbose,exec,cpu,reorder,simple:any,undef,src_f32::blocked:cdba:f0 dst_f32:p:blocked:Acdb16a:f0,,,10x4x3x3,0.187012
-dnnl_verbose,exec,cpu,convolution,jit:avx512_common,forward_training,src_f32::blocked:abcd:f0 wei_f32:p:blocked:Acdb16a:f0 bia_undef::undef::f0 dst_f32:p:blocked:aBcd16b:f0,,alg:convolution_direct,mb4_ic4oc10_ih128oh128kh3sh1dh0ph1_iw128ow128kw3sw1dw0pw1,0.266113
-```
-
-### Run in Batch Mode on DevCloud
-The batch script will activate your environment for each sample and run each sample.
+### Submit The Sample in Batch Modeon
```
Comment (bdmoore1) [inline AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md:128]: Should Modeon be Mode?

File: AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md [hunk 6]
```diff
@@ -121,141 +121,22 @@ dnnl_verbose,exec,cpu,convolution,jit:avx512_common,forward_training,src_f32::bl
...
 ```
 cd ~/oneAPI-samples/AI-and-Analytics/Getting-Started Samples/IntelTensorFlow_GettingStarted
 ```
-
-2.	Create a new file titled hello-world.sh.
-```
-vim hello-world.sh
-```
-3.	Add these two lines to the top of the file:
+2. submit this "TensorFlow_HelloWorld" workload on the selected node with the run script.
```
Comment (bdmoore1) [inline AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md:134]: Capitalize "Submit"

File: AI-and-Analytics/README.md [hunk 7]
```diff
@@ -20,9 +20,27 @@ Third party program Licenses can be found here: [third-party-programs.txt](https
...
 You can use AI Kit samples in
 the [Intel oneAPI DevCloud](https://devcloud.intel.com/oneapi/get-started/) environment in the following ways:
-* Log in to a DevCloud system via SSH and
-  * use `git clone` to get a full copy of samples repository, or
-  * use the `oneapi-cli` tool to download specific sample.
-* Launch a JupyterLab server and run Jupyter Notebooks from your web browser.
+* Log in to a DevCloud system via SSH
+* Launch a JupyterLab server and run Jupyter Notebooks from your web browser.   
+> Please refer to [DevCloud README](DevCloudREADME.md) for more details.
+## Get Samples Codes
```
Comment (bdmoore1) [inline AI-and-Analytics/README.md:29]: Change to ## Get Code Samples

**Review by tomlenth — CHANGES_REQUESTED** [review]
This may seem obvious, but between step 7. "You will see your login at the prompt"  and step 8 about the readme, we need to tell the user to actually get the samples!  It's very disjointed how it is now since they are sitting at a prompt, but there isn't a readme anywhere. ideally we'd provide a link to pull the repo.

File: AI-and-Analytics/DevCloudREADME.md (hunk 1 above)
Comment (louie-tsai) [reply AI-and-Analytics/DevCloudREADME.md:6]: hi Ben. this is the link of your account. we couldn't use that link for all users.

File: AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md (hunk 3 above)
Comment (louie-tsai) [reply AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md:85]: Hi, probably "conda activate pytorch".
Other possible cause is no correct pytorch installation from AI kit.

File: AI-and-Analytics/DevCloudREADME.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+    a.	If you are redirected to the Intel DevCloud page, scroll to the bottom and select Launch Jupyter Lab.
+![](images/jupyter-button.png)
+    b.  When Jupyter Lab opens, if prompted for a kernel, select **No Kernel**.
```
[the remark this answers is not in the document]
Comment (louie-tsai) [reply AI-and-Analytics/DevCloudREADME.md:10]: thanks. addressed it accordingly.

File: AI-and-Analytics/DevCloudREADME.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+5.	Close the Welcome page. The Launcher tab will appear.
+6.  On the Launcher tab, click **Terminal**.
+![](images/jupyter-terminal.png)
+
+7.	You will see your login at the prompt.
+8.	Follow the instructions for DevCloud in the README of each sample
```
[the remark this answers is not in the document]
Comment (louie-tsai) [reply AI-and-Analytics/DevCloudREADME.md]: thanks. addressed it accordingly.

File: AI-and-Analytics/DevCloudREADME.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+ 
+### Running the Sample in DevCloud with a Local Terminal
+1.	Open a terminal on your Linux system.
+2.	Log in to DevCloud.
+```
+ssh devcloud
+```
+3.	Follow the instructions for DevCloud in the README of each sample
```
[the remark this answers is not in the document]
Comment (louie-tsai) [reply AI-and-Analytics/DevCloudREADME.md]: thanks. addressed it accordingly.

File: AI-and-Analytics/End-to-end-Workloads/README.md [hunk 11]
```diff
@@ -27,12 +27,5 @@ Third party program Licenses can be found here:
 | OpenVino           | [LidarObjectDetection-PointPillars](LidarObjectDetection-PointPillars) | Performs 3D object detection and classification using point cloud data from a LIDAR sensor as input.
 
 # Using Samples in the Intel oneAPI DevCloud
+Please refer to [Using AI samples in Intel oneAPI DevCloud](https://github.com/intel-ai-tce/oneAPI-samples/tree/devcloud/AI-and-Analytics#using-samples-in-intel-oneapi-devcloud).
```
[the remark this answers is not in the document]
Comment (louie-tsai) [reply AI-and-Analytics/End-to-end-Workloads/README.md]: thanks. addressed it accordingly.

File: AI-and-Analytics/Features-and-Functionality/README.md [hunk 12]
```diff
@@ -28,10 +28,5 @@ Third party program Licenses can be found here: [third-party-programs.txt](https
 | XGBoost | [IntelPython_XGBoost_daal4pyPrediction](IntelPython_XGBoost_daal4pyPrediction) |  Analyze the performance benefit of minimal code changes to port pre-trained XGBoost model to daal4py prediction for much faster prediction than XGBoost prediction..
 
 # Using Samples in Intel oneAPI DevCloud
+Please refer to [Using AI samples in Intel oneAPI DevCloud](https://github.com/intel-ai-tce/oneAPI-samples/tree/devcloud/AI-and-Analytics#using-samples-in-intel-oneapi-devcloud).
```
[the remark this answers is not in the document]
Comment (louie-tsai) [reply AI-and-Analytics/Features-and-Functionality/README.md]: thanks. addressed it accordingly.

File: AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md (hunk 4 above)
Comment (louie-tsai) [reply AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md]: thanks. addressed it accordingly.

File: AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md (hunk 5 above)
Comment (louie-tsai) [reply AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md]: thanks. addressed it accordingly.

File: AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md (hunk 6 above)
Comment (louie-tsai) [reply AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md]: thanks. addressed it accordingly.

File: AI-and-Analytics/README.md (hunk 7 above)
Comment (louie-tsai) [reply AI-and-Analytics/README.md]: thanks. addressed it accordingly.

File: AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md (hunk 3 above)
Comment (louie-tsai) [reply AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md:85]: have your "source activate pytorch" first?
another possible cause is unsuccessful installation for pytorch from AI kit.

**Comment by louie-tsai** [comment]
> This may seem obvious, but between step 7. "You will see your login at the prompt" and step 8 about the readme, we need to tell the user to actually get the samples! It's very disjointed how it is now since they are sitting at a prompt, but there isn't a readme anywhere. ideally we'd provide a link to pull the repo.  
addressed this feedback with related modification. we will guide them to how to get samples section accordingly.

**Comment by louie-tsai** [comment]
@tomashek @bdmoore1 
please help me to review the contents. 
the main change is adding a Q script for DevCloud job submission.

**Review by JoeOster — APPROVED** [review]

**Review by JoeOster — APPROVED** [review]

**Pull request merged**